etc/kubernetes/manifests/overlays/local/camel-k/integration-platform.yaml

```diff
@@ -1,4 +1,6 @@
 apiVersion: camel.apache.org/v1
 kind: IntegrationPlatform
 metadata:
-  name: camel-k
+  name: cos-fleetshard-operator-camel
+  annotations:
+    camel.apache.org/operator.id: "cos-fleetshard-operator-camel"
```

Comment:
@rinaldodev are these overlays enough to make the changes in prod as well? once the addon generation picks this up?

Reply:
Yes. The staging overlay is used for prod as well.